src/main/java/pl/jkap/sozzt/contract/domain/ContractEntity.java

```diff
@@ -0,0 +1,45 @@
+package pl.jkap.sozzt.contract.domain;
+
+
+import lombok.*;
+import pl.jkap.sozzt.contract.dto.ContractDto;
+import pl.jkap.sozzt.contract.dto.ContractStepEnum;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+@Entity
+@Table(name = "Contract")
+@Getter
+@Builder
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+class ContractEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String invoiceNumber;
+    private String location;
+    private String executive;
+    private LocalDateTime created;
+
+    @Column(name = "contract_step")
+    @Enumerated(EnumType.STRING)
+    private ContractStepEnum contractStepEnum;
+
+    private boolean scanFromTauronUpload;
+
+    ContractDto dto() {
+        return ContractDto.builder()
+                .id(id)
+                .invoiceNumber(invoiceNumber)
+                .location(location)
+                .executive(executive)
+                .scanFromTauronUpload(scanFromTauronUpload)
+                .contactStepEnum(contractStepEnum)
+                .created(created)
+                .build();
+    }
```

Comment:
This should be done in mapper if you use them, do you use mapstruct and mappers in this project?

Reply:
It's nice, but I can't use this (or not for all). For example I have to map ContractEntity to DataInputContract <- these classes have different attributes.

Reply:
Thanks for showing me this! 
https://github.com/szpalajoan/sozzt/blob/MR_fix/src/main/java/pl/jkap/sozzt/contract/domain/ContractMapperInterface.java